### DISCUSSION TITLE
gitlab - create tag without creating releases/uploading files

### DISCUSSION THREAD

**Discussion by swaroopar** [jreleaser/jreleaser#372] [Q&A] [answered]
Hello, 

We are using jreleaser with maven. Is it possible to configure jreleaser to only create tag in gitlab but not create any release in gitlab?

Thank you in advance.
Swaroop

**Comment by aalmiray** [comment, marked as answer]
Hi Swaroop,

I'm afraid not. Currently it's possible to instruct JReleaser to create a release while skipping the tag, as it's assumed the tag was created externally. Releases and tags go hand in hand, you can't have one without the other, at least that's how GitHub/GitLab/Gitea do it at the moment.

Creating just the tag and skip a release is an use case we did not foresee and as such it's not covered by the current implementation. Based on the other discussion topic you raised it seems to me that you;r like to host all release assets (including the generated changelog) in Artifactory, in which case just tagging the repository and skipping the release is desired.

A temporary workaround is to tag the repository yourself, you might do this with an external script (see the antrun or groovy maven plugins). The calculated tag is found at `target/jreleaser/output.properties` plus a few other values that might be useful.

**Reply by swaroopar** [reply]
Thanks @aalmiray for the clarification. This helps.

**Reply by aalmiray** [reply]
Filed https://github.com/jreleaser/jreleaser/issues/376
